controllers/admin/AdminTranslationsController.php

```diff
@@ -2806,7 +2806,9 @@ public function initFormMails($no_display = false)
 
         foreach ($modules_has_mails as $module_name => $module_path) {
             $module_path = rtrim($module_path, '/');
-            $module_mails[$module_name] = $this->getMailFiles($module_path . '/mails/' . $this->lang_selected->iso_code . '/', 'module_mail');
+            $module_locale_mails_path = $module_path . '/mails/' . $this->lang_selected->iso_code . '/';
+            $module_mails[$module_name] = $this->getMailFiles($module_locale_mails_path, 'module_mail');
+            $module_mails[$module_name]['subject'] = $this->getSubjectMailContent($module_locale_mails_path);
             $module_mails[$module_name]['subject'] = $core_mails['subject'];
```

Comment:
isn't this overriding the assignation on the line just above?

Reply:
you're right. thanks